pkg/sink/postgres.go

```diff
@@ -33,23 +33,28 @@ import (
 type PGXSink struct {
 	BaseSink
 
-	ConnStr   string
-	SourceID  string
-	Renice    int64
-	LogReader io.Reader
-
-	conn      *pgx.Conn
-	raw       *pgconn.PgConn
-	schema    *decode.PGXSchemaLoader
-	log       *logrus.Entry
-	prev      cursor.Checkpoint
-	pgSrcID   pgtype.Text
-	pgVersion int64
-	replLag   int64
-	inTX      bool
-	skip      map[string]bool
-	inserts   insertBatch
-	prevDDL   uint32
+	ConnStr     string
+	SourceID    string
+	Renice      int64
+	LogReader   io.Reader
+	BatchTXSize int
+
+	conn             *pgx.Conn
+	raw              *pgconn.PgConn
+	pipeline         *pgconn.Pipeline
+	schema           *decode.PGXSchemaLoader
+	log              *logrus.Entry
+	prev             cursor.Checkpoint
+	pgSrcID          pgtype.Text
+	pgVersion        int64
+	replLag          int64
+	skip             map[string]bool
+	inserts          insertBatch
+	prevDDL          uint32
+	pendingCommitted []cursor.Checkpoint
+	inTX             bool
+	queryParams      []queryParam
```

Comment:
Why this is called `queryParams`? I believe something like `pendingChagnes` would be better.